### DISCUSSION TITLE
Ability to resize the icons, always lit up profile pictures

### DISCUSSION THREAD

**Discussion by jtdy5** [flamewave000/discord-overlay#35] [Ideas]
Would be nice to be able to resize the voice chat icons and names so they're either bigger or smaller.
It would also be nice if the profile pictures were always lit up.

**Comment by flamewave000** [comment]
Unfortunately I don't decide on the UI. Everything you see inside the window is rendered by Discord itself and is the same overlay that is shown in games. You would need to make a twist with them.
